docs/upgrading/upgrading-to-4.3.md

```diff
@@ -1,5 +1,29 @@
 # Rundeck 4.3 Upgrade Notes
 
 :::warning
-Any keys or passwords created using encryption in 4.2.0/4.2.1 will be lost when upgrading, it is recommended to back up your data before upgrading from those versions. Data created in earlier versions (before v4.2.0) is will transfer properly.
-:::
\ No newline at end of file
+Any keys or passwords created using encryption in 4.2.0/4.2.1 will be lost when upgrading, it is recommended to back up your data before upgrading from those versions. Data created in earlier versions (before v4.2.0) will transfer properly.
+:::
+
+## Private keys and password encryption issue
+
+The release 4.3.0 fixes an issue that results in invalid credentials after upgrading from version 3.4.x to version 4.2.0.
+
+This issue is related to the key encryption method and causes keys already stored using previous versions to become invalid.
+
+One of the impacts related to this is related to the private keys used in SCM become invalid disabling all projects that use SCM with SSH private keys.
```

Comment:
Typos

`
This issue is related to the key encryption method and causes keys already stored using previous versions to become invalid.

Specifically, private keys used in SCM becoming invalid after the upgrade. This results in disabling all projects at that use SCM with SSH private keys.`